explorer/interpreter/interpreter.cpp

```diff
@@ -211,10 +211,17 @@ auto Interpreter::EvalPrim(Operator op, Nonnull<const Value*> static_type,
       return heap_.Read(cast<PointerValue>(*args[0]).address(), source_loc);
     case Operator::AddressOf:
       return arena_->New<PointerValue>(cast<LValue>(*args[0]).address());
-    case Operator::Combine:
+    case Operator::BitwiseAnd:
```

Comment:
```suggestion
    case Operator::BitwiseAnd:
      // If & wasn't rewritten, it's being used in a constraint.
```

Just looking for the explanation of why this one is different.

Reply:
```suggestion
    case Operator::BitwiseAnd:
      // If & wasn't rewritten, it's being used to form a constraint.
```
I think this is a bit more precise.